contracts/NodeDelegator.sol

```diff
@@ -172,37 +180,65 @@ contract NodeDelegator is INodeDelegator, LRTConfigRoleChecker, PausableUpgradea
             IEigenStrategyManager(eigenlayerStrategyManagerAddress).getDeposits(address(this));
 
         uint256 strategiesLength = strategies.length;
-        assets = new address[](strategiesLength);
-        assetBalances = new uint256[](strategiesLength);
+        // Add 1 to length to include WETH
+        assets = new address[](strategiesLength + 1);
+        assetBalances = new uint256[](strategiesLength + 1);
 
         for (uint256 i = 0; i < strategiesLength;) {
             assets[i] = address(IStrategy(strategies[i]).underlyingToken());
+            // Get LST balance in EigenLayer strategies
             assetBalances[i] = IStrategy(strategies[i]).userUnderlyingView(address(this));
+            // Get LST balance in this NodeDelegator contract
+            assetBalances[i] += IERC20(assets[i]).balanceOf(address(this));
             unchecked {
                 ++i;
             }
         }
-    }
 
-    /// @dev Returns the balance of an asset that the node delegator has deposited into the strategy
-    /// @param asset the asset to get the balance of
-    /// @return stakedBalance the balance of the asset
-    function getAssetBalance(address asset) external view override returns (uint256) {
-        address strategy = lrtConfig.assetStrategy(asset);
-        if (strategy == address(0)) {
-            return 0;
+        // The WETH address is different between chains so reading from the config.
+        // Ideally, the WETH address would be an immutable but following the existing pattern of using config for now.
+        assets[strategiesLength] = lrtConfig.getLSTToken(LRTConstants.WETH_TOKEN);
+        // Sum up this NodeDelegator contract's WETH balance, ETH balance and ETH staked but not verified
+        assetBalances[strategiesLength] +=
```

Comment:
This might be clearer using a temp variable for the WETH token address.

Reply:
I went with not adding an extra line setting a `WETH_TOKEN_ADDRESS` variable thinking the gas wasn't that different but it turns out its more than a trivial amount.

Before
```
| getAssetBalances                                   | 41554           | 41554  | 41554  | 41554  | 1       |
```
After adding `WETH_TOKEN_ADDRESS` variable
```
| getAssetBalances                                   | 41481           | 41481  | 41481  | 41481  | 1       |
```